h/static/scripts/annotator/plugin/test/pdf-metadata-test.js

```diff
@@ -0,0 +1,73 @@
+'use strict';
+
+var PDFMetadataService = require('../pdf-metadata');
```

Comment:
The name here should match the class name if possible, so just `PDFMetadata` rather than `PDFMetadataService`.